### PR TITLE
mandala deployment

### PR DESCRIPTION
**Pull request by frankyin-factual**

### CODE REVIEW

File: primitives/src/currency.rs [hunk 1]
```diff
@@ -184,6 +184,7 @@ create_currency_id! {
 		AUSD("Acala Dollar", 12) = 1,
 		DOT("Polkadot", 10) = 2,
 		LDOT("Liquid DOT", 10) = 3,
+		TAI("Taiga", 12) = 4,
 		// 20 - 39: External tokens (e.g. bridged)
```
Comment (xlc) [inline primitives/src/currency.rs:188]: move this to 132, it is part of the Karura & Kusama token

File: runtime/mandala/src/lib.rs [hunk 2]
```diff
@@ -789,7 +789,8 @@ parameter_type_with_key! {
 				TokenSymbol::ACA |
 				TokenSymbol::KINT |
 				TokenSymbol::KBTC |
-				TokenSymbol::CASH => Balance::max_value() // unsupported
+				TokenSymbol::TAI |
+				TokenSymbol::CASH => 10 * millicent(*currency_id) // unsupported
```
Comment (xlc) [inline runtime/mandala/src/lib.rs:793]:
```suggestion
				TokenSymbol::CASH => 10 * millicent(*currency_id)
```

File: runtime/karura/src/lib.rs [hunk 3]
```diff
@@ -754,6 +754,7 @@ parameter_type_with_key! {
 				TokenSymbol::PHA => 4000 * millicent(*currency_id), // 400PHA = 1KSM
 				TokenSymbol::KINT => 13333 * microcent(*currency_id), // 1.33 KINT = 1 KSM
 				TokenSymbol::KBTC => 66 * microcent(*currency_id), // 1KBTC = 150 KSM
+				TokenSymbol::KAI => 10 * millicent(*currency_id),
```
Comment (xlc) [inline runtime/karura/src/lib.rs:757]: move this to unsupported in this PR. we can enable it in later PR

**Review by xlc — CHANGES_REQUESTED** [review]

File: primitives/src/currency.rs (hunk 1 above)
Comment (frankyin-factual) [reply primitives/src/currency.rs:188]: got it

**Comment by frankyin-factual** [comment]
fixed

File: runtime/karura/src/lib.rs [hunk 4]
```diff
@@ -760,6 +760,7 @@ parameter_type_with_key! {
 				TokenSymbol::DOT |
 				TokenSymbol::LDOT |
 				TokenSymbol::RENBTC |
+				TokenSymbol::KAI |
```
Comment (xlc) [inline runtime/karura/src/lib.rs:763]:
```suggestion
				TokenSymbol::TAI |
```

**Review by xlc — APPROVED** [review]

File: runtime/mandala/src/lib.rs [hunk 5]
```diff
@@ -789,7 +789,8 @@ parameter_type_with_key! {
 				TokenSymbol::ACA |
 				TokenSymbol::KINT |
 				TokenSymbol::KBTC |
-				TokenSymbol::CASH => Balance::max_value() // unsupported
+				TokenSymbol::TAI |
+				TokenSymbol::CASH => 10 * millicent(*currency_id)
```
Comment (zjb0807) [inline runtime/mandala/src/lib.rs:793]:
```suggestion
				TokenSymbol::CASH => Balance::max_value() // unsupported
```

move `TokenSymbol::TAI` below `TokenSymbol::PHA`

**Review by zjb0807 — APPROVED** [review]

**Pull request merged**